pubsublite/ps/doc.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+/*
+Package ps contains a publisher and subscriber client for the Cloud Pub/Sub Lite
+service that emulates the Cloud Pub/Sub API.
+
+If interfaces are defined for pubsub.Topic.Publish() and
+pubsub.Subscription.Receive(), the clients in this package can be used as
+drop-in replacements. As noted in comments, the two services have some
+differences:
+
+- Pub/Sub Lite does not support nack for messages.
```

Comment:
I would also mention that Lite has no concept of "ack expiration;" you **must** call Ack on every message you receive, or shut down the subscriber client.

Reply:
Done